This is not in the original scope of this PR, but it would be good to fix these warnings when you are at it:
```
In file included from /Users/.../src/gridkit/GridKit/src/Model/PhasorDynamics/Load/Load.cpp:2:
/Users/.../src/gridkit/GridKit/src/Model/PhasorDynamics/Load/LoadImpl.hpp:107:40: warning: 'always_inline' function might not be inlinable unless also declared 'inline' [-Wattributes]
  107 |     __attribute__((always_inline)) int Load<ScalarT, IdxT>::evaluateResidualLocally(ScalarT* y, [[maybe_unused]] ScalarT* yp, ScalarT* f)
      |                                        ^~~~~~~~~~~~~~~~~~~
/Users/.../src/gridkit/GridKit/src/Model/PhasorDynamics/Load/LoadImpl.hpp:107:40: warning: 'always_inline' function might not be inlinable unless also declared 'inline' [-Wattributes]
In file included from /Users/55y/src/gridkit/GridKit/src/Model/PhasorDynamics/Load/LoadDependencyTracking.cpp:2:
/Users/.../src/gridkit/GridKit/src/Model/PhasorDynamics/Load/LoadImpl.hpp:107:40: warning: 'always_inline' function might not be inlinable unless also declared 'inline' [-Wattributes]
  107 |     __attribute__((always_inline)) int Load<ScalarT, IdxT>::evaluateResidualLocally(ScalarT* y, [[maybe_unused]] ScalarT* yp, ScalarT* f)
      |                                        ^~~~~~~~~~~~~~~~~~~
/Users/.../src/gridkit/GridKit/src/Model/PhasorDynamics/Load/LoadImpl.hpp:107:40: warning: 'always_inline' function might not be inlinable unless also declared 'inline' [-Wattributes]
In file included from /Users/.../src/gridkit/GridKit/src/Model/PhasorDynamics/SynchronousMachine/GenClassical/GenClassicalDependencyTracking.cpp:2:
/Users/.../src/gridkit/GridKit/src/Model/PhasorDynamics/SynchronousMachine/GenClassical/GenClassicalImpl.hpp:190:40: warning: 'always_inline' function might not be inlinable unless also declared 'inline' [-Wattributes]
  190 |     __attribute__((always_inline)) int GenClassical<ScalarT, IdxT>::evaluateResidualLocally(ScalarT* y, ScalarT* yp, ScalarT* f)
      |                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/.../src/gridkit/GridKit/src/Model/PhasorDynamics/SynchronousMachine/GenClassical/GenClassicalImpl.hpp:190:40: warning: 'always_inline' function might not be inlinable unless also declared 'inline' [-Wattributes]
In file included from /Users/.../src/gridkit/GridKit/src/Model/PhasorDynamics/SynchronousMachine/GenClassical/GenClassical.cpp:2:
/Users/.../src/gridkit/GridKit/src/Model/PhasorDynamics/SynchronousMachine/GenClassical/GenClassicalImpl.hpp:190:40: warning: 'always_inline' function might not be inlinable unless also declared 'inline' [-Wattributes]
  190 |     __attribute__((always_inline)) int GenClassical<ScalarT, IdxT>::evaluateResidualLocally(ScalarT* y, ScalarT* yp, ScalarT* f)
      |                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/.../src/gridkit/GridKit/src/Model/PhasorDynamics/SynchronousMachine/GenClassical/GenClassicalImpl.hpp:190:40: warning: 'always_inline' function might not be inlinable unless also declared 'inline' [-Wattributes]
```

--- a/src/Model/PhasorDynamics/Exciter/IEEET1/Ieeet1Impl.hpp
+++ b/src/Model/PhasorDynamics/Exciter/IEEET1/Ieeet1Impl.hpp
@@ -142,7 +142,7 @@ namespace GridKit
         // other variables.
         if (signals_.template isAssigned<Ieeet1InternalVariables::EFD>())
         {
-          efd0 = y_[7]; ///<- generator needs to be initialized first
+          efd0 = y_[7]; //<- generator needs to be initialized first
         }
 
         // Terminal Voltage

--- a/src/Model/PhasorDynamics/Governor/Tgov1/Tgov1Impl.hpp
+++ b/src/Model/PhasorDynamics/Governor/Tgov1/Tgov1Impl.hpp
@@ -160,7 +160,7 @@ namespace GridKit
         // Initial mechanical = initial electric torque
         if (signals_.template isAssigned<Tgov1InternalVariables::PM>())
         {
-          p0 = y_[2]; ///<- generator needs to be initialized first
+          p0 = y_[2]; //<- generator needs to be initialized first
         }
 
         // Input Variables (Parameter for now)

--- a/src/Model/PhasorDynamics/Load/Load.hpp
+++ b/src/Model/PhasorDynamics/Load/Load.hpp
@@ -95,7 +95,7 @@ namespace GridKit
       }
 
     public:
-      __attribute__((always_inline)) int evaluateResidualLocally(ScalarT*, ScalarT*, ScalarT*);
+      __attribute__((always_inline)) inline int evaluateResidualLocally(ScalarT*, ScalarT*, ScalarT*);
 
     private:
       bus_type* bus_{nullptr};

--- a/src/Model/PhasorDynamics/SynchronousMachine/GENROUwS/CMakeLists.txt
+++ b/src/Model/PhasorDynamics/SynchronousMachine/GENROUwS/CMakeLists.txt
@@ -6,7 +6,7 @@ if(GRIDKIT_ENABLE_ENZYME)
     LINK_LIBRARIES
       PUBLIC GRIDKIT::phasor_dynamics_signal ClangEnzymeFlags
     COMPILE_OPTIONS
-      -mllvm -enzyme-auto-sparsity=1 -fno-math-errno
+      -mllvm -enzyme-auto-sparsity=1 -fno-math-errno -fno-vectorize
     OUTPUT_NAME
       gridkit_phasor_dynamics_genrou)
 else()

--- a/src/Model/PhasorDynamics/SynchronousMachine/GenClassical/GenClassical.hpp
+++ b/src/Model/PhasorDynamics/SynchronousMachine/GenClassical/GenClassical.hpp
@@ -106,7 +106,7 @@ namespace GridKit
       }
 
     public:
-      __attribute__((always_inline)) int evaluateResidualLocally(ScalarT*, ScalarT*, ScalarT*);
+      __attribute__((always_inline)) inline int evaluateResidualLocally(ScalarT*, ScalarT*, ScalarT*);
 
     private:
       /* Identification */

